addons/explosives/functions/fnc_defuseExplosive.sqf

```diff
@@ -17,7 +17,8 @@
 #include "script_component.hpp"
 EXPLODE_2_PVT(_this,_unit,_explosive);
 
-if (getNumber (ConfigFile >> "CfgAmmo" >> typeof _explosive >> "ACE_explodeOnDefuse") == 1) exitWith {
+_explode = random 1.0;
```

Comment:
This variable is unnecessary (and missing a private); you can use the `random 1` straight into the if statement below.
```sqf
if (GVAR(ExplodeOnDefuse) && (random 1) < getNumber(ConfigFile >> "CfgAmmo" >> typeOf _explosive >> "ACE_explodeOnDefuse")) exitWith {
```